commands/deploy.go

```diff
@@ -261,7 +262,7 @@ func newFileUploadRequest(c *DeployCmd, f *os.File) (r *http.Request, err error)
 	}
 	defer f.Close()
 
-	body := new(bytes.Buffer)
+	body := &bytes.Buffer{}
 	writer := multipart.NewWriter(body)
```

Comment:
`writer := multipart.NewWriter(&body)`

Reply:
I was getting some weird behavior before when I was trying this, but it appears to be working fine. 

Fixed in 1bb4712b1a1f3a73dcb191c03340c6fcc9d10ac8.